hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestBucketPut.java

```diff
@@ -65,8 +65,8 @@ public void testBucketFailWithAuthHeaderMissing() throws Exception {
     try {
       bucketEndpoint.put(bucketName, null, null, null);
     } catch (OS3Exception ex) {
-      Assert.assertEquals(HTTP_NOT_FOUND, ex.getHttpCode());
-      Assert.assertEquals(MALFORMED_HEADER.getCode(), ex.getCode());
+      assertEquals(HTTP_NOT_FOUND, ex.getHttpCode());
```

Comment:
nit: please use assertThrow().

Reply:
Done.

Reply:
This test is not doing anything if I'm not wrong because it will never reach to catch. If I'm right, please create a jira to fix it. Same for `testBucketFailWithInvalidHeader`.

Reply:
[HDDS-9760](https://issues.apache.org/jira/browse/HDDS-9760)